src/index.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-expressions */
+import display from './display.js';
+import './styles.css';
+import { postData } from './main.js';
+
+const formInput = document.getElementById('my-form');
+const refresh = document.getElementById('refresh');
+
+document.addEventListener('DOMContentLoaded', () => {
+  refresh.addEventListener('click', async () => {
+    display();
+  });
+
+  formInput.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    const playerName = document.getElementById('username').value;
+    const outcome = document.getElementById('score').value;
+    postData();
+    playerName === '';
+    outcome === '';
+  });
```

Comment:
[OPTIONAL]

- After submitting the form with the `name` and `score`, it's better to make the field empty for a better UI.